### DISCUSSION TITLE
Exporting blockIODevice in version 1.1.4

### DISCUSSION THREAD

**Discussion by m-kat** [opencontainers/runc#3649] [Q&A]
Hello, 
there is a commit in main (c0be1aa2d101dcd3074b5a0e486d58d3f9568d81) that exports the struct BlockIODevice in file libcontainer/configs/blkio_device.go . I'm currently using podman 4.3 as a library to embed some of the functionality in the new tool. Podman v4.3 depends on opencontainer/runc v1.1.4 when resolved by go modules, but in order to compile it, the commit mentioned above is needed. Unfortunately whenever golang tries to resolve that commit from runc, it sees it as v1.1.1-0.20220613174039-c0be1aa2d101, so it connects it to v1.1.1 . Is it possible to cherry-pick the change to the v1.1.4 in order to resolve that dependency? 
Thank you in advance for your fast response.

**Comment by AkihiroSuda** [comment]
I'd rather prefer to release v1.2.0 ASAP. 

> v1.1.1-0.20220613174039-c0be1aa2d101, so it connects it to v1.1.1 . 

No, this points to https://github.com/opencontainers/runc/commit/c0be1aa2d101dcd3074b5a0e486d58d3f9568d81 in the `main` branch.
The "v1.1.1-" prefix means that the `release-1.1` branch for the `v1.1.x` releases have diverged from the `main` branch since `v1.1.1`.

**Comment by kolyshkin** [comment]
The preliminary plan is to release 1.2.0 this month. This looks a bit ambitious but let's try.

Backporting this to 1.1.x is a less desirable option, since we mostly backport fixes and this looks more like a feature.
